### ISSUE TITLE
Registrations on clean install

### ISSUE THREAD

**Issue by samael33** [fusionpbx/fusionpbx#1051]
Extensions can't register on internal profile

   ------------------------------------------------------------------------
recv 418 bytes from udp/[176.37.91.15]:5060 at 12:18:43.982942:
   ------------------------------------------------------------------------
   REGISTER sip:pbx.ilaber.com SIP/2.0
   Via: SIP/2.0/UDP 192.168.88.51:5060;branch=z9hG4bK.MsalqAFKl;rport
   From: <sip:600@pbx.ilaber.com>;tag=OPZLBxwcO
   To: sip:600@pbx.ilaber.com
   CSeq: 24 REGISTER
   Call-ID: eV7cmSZdaT
   Max-Forwards: 70
   Supported: outbound
   Contact: <sip:600@176.37.91.15>;+sip.instance="<urn:uuid:c90c96b3-8419-4fb6-88c8-6c49fdac6497>"
   Expires: 3600
   User-Agent: Linphone/3.8.0 (belle-sip/1.4.0)

   ------------------------------------------------------------------------
send 416 bytes to udp/[176.37.91.15]:5060 at 12:18:43.983262:
   ------------------------------------------------------------------------
   SIP/2.0 405 Method Not Allowed
   v:SIP/2.0/UDP 192.168.88.51:5060;branch=z9hG4bK.MsalqAFKl;rport=5060;received=176.37.91.15
   From: <sip:600@pbx.ilaber.com>;tag=OPZLBxwcO
   t:sip:600@pbx.ilaber.com;tag=604N1jvBX777g
   Call-ID: eV7cmSZdaT
   CSeq: 24 REGISTER
   User-Agent:FreeSWITCH Rocks!
   Allow:INVITE,ACK,BYE,CANCEL,OPTIONS,MESSAGE,INFO,UPDATE,PRACK,NOTIFY,PUBLISH,SUBSCRIBE
   k:precondition,100rel,path,replaces
   l:0

How to reproduce: 
Install fusion, make a domain with FQDN, delete domain with IP.
Make an extension on domain, try to register.

Workaround - change context of internal profile to "default"

Same issue - http://serverfault.com/questions/703698/fusionpbx-sip-2-0-405-method-not-allowed-sip-phones-not-registering

**Comment by richbreton** [comment]
did you install this from git 
https://raw.githubusercontent.com/fusionpbx/fusionpbx-scripts/master/install/ubuntu/install_fusionpbx.sh

?

**Comment by richbreton** [comment]
if you installed from google code you are probably experiencing a bug from that release that was fixed with the github verson. The github version is not a release but its more stable than google code 3.8

**Comment by samael33** [comment]
Yea, I've installed from google code (one, that in home page link). Will try to play with this one version. Thank you.

**Comment by markjcrane** [comment]
Anyone using google code will have to migrate to GIT at some point.
http://wiki.fusionpbx.com/index.php?title=Main_Page#Migration_from_SVN_to_GIT

**Comment by richbreton** [comment]
if your not too far in you can start from scratch reading this page:

https://github.com/fusionpbx/fusionpbx

**Comment by richbreton** [comment]
@samael33 let us know what you come up with, I want to close the issue ;)

**Comment by samael33** [comment]
Yea, for now it registers. But context is still 'public' ) If this is correct - than OK. And seems git version switches to database-driven users and dialplan. If this is correct too - double times OK )

**Comment by anis-bedhiafi** [comment]
I used the script install google code and I managed to fix it by doing the following:
-Go to Advanced > SIP Profiles
-Click on "Internal"
-Find "bind-params". You'll notice that there is no tcp transport, only udp is there. Or registrations use the TCP transport protocol.
-Go ahead and click on "bind-params" and and tcp to its value. like this (transport=udp,tcp).
-Save
-Now go to Status > SIP Status and restart the "sofia status profile internal"
- Try registring your phones 
-Done :)
